pkg/samples/samples.go

```diff
@@ -233,6 +235,47 @@ func (s *Samples) Copy(target string) error {
 	return nil
 }
 
+func (s *Samples) ConfigureDotEnv(sampleLocation string) error {
```

Comment:
We hardcode some of the .env paths in the servers so I think we'd have a similar problem to the STATIC_DIR path problem when users pick a single language. I'm updating the servers to hardcode the paths in a ENV_PATH variable, so if you look for that and remove one of the "../" that should probably work.